e2e/testapp/polard/cmd/root.go

```diff
@@ -161,7 +162,14 @@ func NewRootCmd() *cobra.Command {
 // initCometBFTConfig helps to override default CometBFT Config values.
 // return cmtcfg.DefaultConfig if no custom configuration is required for the application.
 func initCometBFTConfig() *cmtcfg.Config {
-	return cmtcfg.DefaultConfig()
+	cfg := cmtcfg.DefaultConfig()
+	consensus := cfg.Consensus
+	consensus.TimeoutCommit = time.Second * 2
+	consensus.TimeoutPropose = time.Second * 2
+	consensus.TimeoutPrevote = time.Second * 2
+	consensus.TimeoutPrecommit = time.Second * 2
+	consensus.TimeoutCommit = time.Second * 2
```

Comment:
thes should be read in from a file?

Reply:
just setting defaults to prevent having to do the `jq` thing